src/jmespath.net/jmespath.net.csproj

```diff
@@ -26,13 +26,13 @@
 
   <ItemGroup Condition="'$(TargetFramework)' == 'netstandard1.3'">
     <PackageReference Include="System.Reflection.TypeExtensions" Version="4.7.0" />
-	<PackageReference Include="System.ValueTuple" Version="4.5.0" />
+    <PackageReference Include="System.ValueTuple" Version="4.5.0" />

```

Comment:
Likewise, please do not mix concerns in a single pull request.

Reply:
Will look at reverting the whitespace tomorrow.